Makefile

```diff
@@ -92,6 +92,19 @@ test-cpp-omp:
 	cmake --build ./BuildTests --target runner
 	cmake --build ./BuildTests --target test
 
+test-cpp-kokkos:
+	rm -rf ./BuildTests
+	cmake $(LIGHTNING_CPP_DIR) -BBuildTests -DBUILD_TESTS=ON -DENABLE_KOKKOS=ON
+	cmake --build ./BuildTests --target runner
+	cmake --build ./BuildTests --target test
+
+.PHONY: benchmark
+benchmark:
+	cmake --build BuildBench --target clean || true
+	rm -rf ./BuildBench/CMakeCache.txt ./BuildBench/compiler_info.txt ./BuildBench/run_gate_benchmark.sh
+	cmake $(LIGHTNING_CPP_DIR) -BBuildBench -DBUILD_EXAMPLES=ON -DCMAKE_BUILD_TYPE=Release -DENABLE_AVX=ON
+	cmake --build ./BuildBench
+
```

Comment:
This part is already removed from Makefile (#274). There is no `run_gate_benchmark.sh` file anymore.